src/ami-set.js

```diff
@@ -0,0 +1,23 @@
+let base = require('taskcluster-base');
+
+const KEY_CONST = 'ami-set';
+
+ /**
+ * An AMI Set is a collection of AMIs with a single name (its AMI Set ID).
+ * Each AMI in the set is keyed by its virtualization type (PV or HVM) and
+ * by its AWS region.
+ */
+
+let amiSet = base.Entity.configure({
+  version: 1,
+
+  partitionKey: base.Entity.keys.ConstantKey(KEY_CONST),
+  rowKey: base.Entity.keys.StringKey('amiSetId'),
+
+  properties: {
+
+    amiSetId: base.Entity.types.String,
+    amis: base.Entity.types.JSON,
+
+  },
+});
```

Comment:
The content of this module looks OK, but it doesn't export anything.  There's some information on exports here:

  https://www.sitepoint.com/understanding-module-exports-exports-node-js/

In this case, we want `require('./ami-set')` to return the configured entity, just like for `require('./worker-type')`.